crowdfunding/crowdfunding/urls.py

```diff
@@ -14,8 +14,22 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
+from django.http import JsonResponse
+from rest_framework.authtoken.views import obtain_auth_token
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('api-auth/', include('rest_framework.urls')),
+    path('api-token-auth/', obtain_auth_token, name='api_token_auth'),
+    path('users/', include('users.urls')),
+    path("", include('projects.urls')),
 ]
+
+def blanket_404_response(request, exception):
+        return JsonResponse({
+            'status_code': 404,
+            'error': "Oops, something went wrong!"
+        })
+
+handler404 = blanket_404_response
```

Comment:
nice 404

Reply:
Thank you for all the feedback I appreciate it heaps :) didn't feel like I did too well at time of submission so it's really encouraging.